bash.html

```diff
@@ -81,11 +86,24 @@ <h1 class=cf>bashfill</h1><h1 class=rf>@</h1><h1 class=gf>MAX.IO</h1><h1 class=y
 
 img="\
 <div id="output"></div>
+
+line=""
 for n in $img
 do
-    u $n
+    if [ "$n" = "e1" ]; then
+        for i in `seq 1 $height`; do
+            echo -n "$line"
+            e
+        done
+        line=""
+    else
+      line="${line}$(u "$n")";
+    fi
+done
+for i in `seq 1 $height`; do
+    echo -n "$line"
+    e
 done
-e;
```

Comment:
this last for loop (or your last e) could be removed if the image always ended in an `e1`.